tensorflow/python/ops/histogram_ops.py

```diff
@@ -88,6 +88,9 @@ def histogram_fixed_width_bins(values,
     # map tensor values within the open interval value_range to {0,.., nbins-1},
     # values outside the open interval will be zero or less, or nbins or more.
     indices = math_ops.floor(nbins_float * scaled_values, name='indices')
+    
+    if array_ops.where(value_range[0] == value_range[1]):
```

Comment:
Use tf.equal instead of == here to make it work in tf1 too.

Also use tf.cond instead of if so it doesn't require autograph

Reply:
@alextp Thanks for your help. This is my first code change. Thanks for taking time out of your busy schedule to help me.